internal/decryptor/decryptor.go

```diff
@@ -187,6 +187,14 @@ func New(client client.Client, kustomization *kustomizev1.Kustomization, opts ..
 	return d, cleanup, nil
 }
 
+// IsDecryptionDisabled checks if the given object has the decrypt: disabled annotation set
+func IsDecryptionDisabled(annotations map[string]string) bool {
+	if annotations != nil && annotations[fmt.Sprintf("%s/decrypt", kustomizev1.GroupVersion.Group)] == kustomizev1.DisabledValue {
+		return true
+	}
```

Comment:
This should be case insensitive, see how we do it here: https://github.com/fluxcd/pkg/blob/0fe2a7c7e3ddedf39d862c98df89f5878777e748/ssa/utils/meta.go#L59